cmd/clusterctl/pkg/client/cluster/cert_manager.go

```diff
@@ -161,6 +193,24 @@ func (cm *certManagerClient) getWebHook(c client.Client) (*unstructured.Unstruct
 	return webHook, nil
 }
 
+// hasWebHook returns true if there is already a web-hook in the cluster
+func (cm *certManagerClient) hasWebHook() (bool, error) {
```

Comment:
```suggestion
func (cm *certManagerClient) hasWebhook() (bool, error) {
```

Reply:
Very minor

Reply:
The same change applies to many other places, so I'm sending a separated PR with s/WebHook/Webhook